pkg/apis/kops/cluster.go

```diff
@@ -317,8 +317,12 @@ type ClusterSubnetSpec struct {
 	CIDR string `json:"cidr,omitempty"`
 
 	// ProviderID is the cloud provider id for the objects associated with the zone (the subnet on AWS)
-	ProviderID string `json:"id,omitempty"`
+	ProviderID string `json:"subnetId,omitempty"`
```

Comment:
Is there a reason to change this from `id`?  I would ask that we don't unless we have to, but I don't know if we have to.